src/test/kotlin/io/sip3/salto/ce/rtcp/RtcpHandlerTest.kt

```diff
@@ -111,6 +111,48 @@ class RtcpHandlerTest : VertxTest() {
                 0x70.toByte(), 0x1a.toByte(), 0x03.toByte(), 0xd7.toByte(),
                 0x00.toByte(), 0x04.toByte(), 0x85.toByte(), 0x1f.toByte()
         )
+
+        val PACKET_4 = byteArrayOf(
```

Comment:
I think this PACKET_4 can be just a string with JSON instead of byte array representation... We have PACKET_1 as a byte array because it's binary.